qcfractal/queue/managers.py

```diff
@@ -469,6 +547,33 @@ def update(self, new_tasks: bool = True, allow_shutdown=True) -> bool:
         # Get new tasks
         payload = self._payload_template()
         payload["data"]["limit"] = open_slots
+
+        # Crunch Statistics
+        self.statistics.total_failed_tasks += n_fail
+        self.statistics.total_successful_tasks += n_success
+        self.statistics.total_core_hours += task_cpu_hours
+        try:
```

Comment:
Can we do this via an `if`?

Reply:
Sure, I do this twice so I'll switch it, any particular reason or just preference of `if...else` over `try...except`?

Reply:
It just seems a bit cleaner and more inline with the rest of the code base. I have often found exception catching to be imprecise for these catches.